scene/gui/item_list.cpp

```diff
@@ -372,7 +371,7 @@ void ItemList::clear() {
 }
 
 void ItemList::set_fixed_column_width(int p_size) {
-
+	ERR_EXPLAIN("fixed_column_width can not be less than 0.")
```

Comment:
Please capitalize the first letter in those `ERR_EXPLAIN` messages (here, 398, and 410).

Reply:
The variable name should likely not be used here then. Somehow like "The fixed column width cannot be less than 0."

Reply:
I actually completely missed that this was a variable name. But I agree that the variable should be ditched from the message.